pkg/handlers/auth.go

```diff
@@ -20,7 +22,14 @@ const (
 )
 
 // TODO: get this dynamically from the platform via SDK or dialing directly: [https://github.com/opentdf/platform/issues/147]
-const TOKEN_URL = "http://localhost:8888/auth/realms/opentdf/protocol/openid-connect/token"
+var TOKEN_URL = "http://localhost:8888/auth/realms/opentdf/protocol/openid-connect/token"
+
+func init() {
+	tokenURL := os.Getenv("OTDFCTL_TOKEN_URL")
+	if tokenURL != "" {
+		TOKEN_URL = tokenURL
+	}
+}
```

Comment:
@jrschumacher This is still temporary but wanted the ability to overwrite it for now.